### PR TITLE
renamed clickSentButton to clickSendButton, updated test after change…

### LINKED ISSUE #1049: `clickSentButton` should be called `clickSendButton`
Should be called `clickSendButton` - filing another issue

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/pull/1030#discussion_r753676950_

### PR DESCRIPTION
**Pull request by fcvakintos**
…s, added fixes for checkEmail in message

This PR contains changes for checking sender email, renamed clickSentButton to clickSendButton

close #1049 

----------------------------------

**Tests** 
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: appium/tests/screenobjects/email.screen.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ class EmailScreen extends BaseScreen {
   }
 
   checkEmailAddress = async (email: string) => {
-    const selector = `~${email}`;
+    const selector = `-ios class chain:**/XCUIElementTypeStaticText[\`label == "${email}"\`]`;
```
Comment (tomholub) [inline appium/tests/screenobjects/email.screen.ts:55]: Whenever you see yourself using this kind of complicated selector, please add accessibility identifier instead, and query by that.

**Review by tomholub — CHANGES_REQUESTED** [review]
Thank you - see comment.

File: appium/tests/screenobjects/email.screen.ts (hunk 1 above)
Comment (fcvakintos) [reply appium/tests/screenobjects/email.screen.ts:55]: will try

File: appium/tests/screenobjects/email.screen.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ class EmailScreen extends BaseScreen {
   }
 
   checkEmailAddress = async (email: string) => {
-    const selector = `-ios class chain:**/XCUIElementTypeStaticText[\`label == "${email}"\`]`;
+    const selector = `~${email}`;
     await (await $(selector)).waitForDisplayed();
```
Comment (tomholub) [inline appium/tests/screenobjects/email.screen.ts]: What I meant is, an accessibility identifier should be added to the sender element on message screen in Swift. Then query here by that identifier.

Comment (tomholub) [reply appium/tests/screenobjects/email.screen.ts]: eg 

```
Error: element ("~e2e.enterprise.test@flowcrypt.com") still not displayed after 15000ms
    at async EmailScreen.checkEmailAddress (/Users/tom/git/flowcrypt-ios/appium/tests/screenobjects/email.screen.ts:56:5)
    at async EmailScreen.checkOpenedEmail (/Users/tom/git/flowcrypt-ios/appium/tests/screenobjects/email.screen.ts:71:5)
    at async UserContext.<anonymous> (/Users/tom/git/flowcrypt-ios/appium/tests/specs/composeEmail/SendEncryptedEmailAfterPassPhraseSessionEndedAndTrashIt.spec.ts:57:5)
```

When in fact I see it on screen. By using accessibility identifiers instead, we remove uncertainty.

Comment (fcvakintos) [reply appium/tests/screenobjects/email.screen.ts]: added senderEmail accessibilityId

**Comment by fcvakintos** [comment]
@tomholub it is failed because we didn't push new accessibilityId in master
https://flowcrypt.semaphoreci.com/jobs/fd396163-6d60-47d2-9b8f-c4b482b9b6a1

File: FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift [hunk 3]
```diff
@@ -74,7 +74,10 @@ public final class ThreadMessageSenderCellNode: CellNode {
         automaticallyManagesSubnodes = true
 
         senderNode.attributedText = input.sender
+        senderNode.accessibilityIdentifier = "senderEmail"
         dateNode.attributedText = input.date
+        senderNode.accessibilityIdentifier = "date"
+
```
Comment (tomholub) [inline FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift:80]: here you meant `dateNode.acc..`

**Review by tomholub — CHANGES_REQUESTED** [review]

**Comment by tomholub** [comment]
> @tomholub it is failed because we didn't push new accessibilityId in master https://flowcrypt.semaphoreci.com/jobs/fd396163-6d60-47d2-9b8f-c4b482b9b6a1

Can you be more specific - which PR needs to be merged for this to pass?

File: FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift (hunk 3 above)
Comment (fcvakintos) [reply FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift:80]: no, 
`senderNode.accessibilityIdentifier = "senderEmail"`

Comment (tomholub) [reply FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift:80]: Please carefully re-read the code I quoted.

**Comment by fcvakintos** [comment]
@tomholub we need to merge this selector 
senderNode.accessibilityIdentifier = "senderEmail"

**Comment by tomholub** [comment]
> @tomholub we need to merge this selector senderNode.accessibilityIdentifier = "senderEmail"

We don't. Appium tests are not testing master (hopefully!!), they are testing the code in this PR. If you just changed an accessibility identifier, it must be directly reflected in tests that run in this PR.

File: FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift (hunk 3 above)
Comment (fcvakintos) [reply FlowCryptUI/Cell Nodes/ThreadMessageSenderCellNode.swift:80]: yes, sorry, fixed already, 
will push

**Review by tomholub — APPROVED** [review]

**Pull request merged**